backend/btrixcloud/crawls.py

```diff
@@ -109,6 +111,11 @@ class Crawl(CrawlConfigCore):
 
     stopping: Optional[bool] = False
 
+    collections: Optional[List[UUID4]] = []
+
+    # Auto-added collections tracked here until crawl successfully finishes
+    autoAddCollections: Optional[List[UUID4]] = []
```

Comment:
What about just querying the workflow when crawl completes and then adding to collections list all in one pass?

One advantage of that is user will be able to watch a running crawl, see if it looks reasonable, and then change the auto-add collections while crawl is still running, since the auto-add happens on crawl complete, while this would still keep the old list.

Reply:
Yeah that works! will change